src/sentry/workflow_engine/condition_handlers/group_event.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any
+
+from sentry.eventstore.models import GroupEvent
+from sentry.workflow_engine.models.data_condition import Condition
+from sentry.workflow_engine.registry import condition_handler_registry
+from sentry.workflow_engine.types import DataConditionHandler
+
+
+def get_nested_value(data: GroupEvent, path: str, default: Any = None) -> Any | None:
+    try:
+        value = data
+        for part in path.split("."):
```

Comment:
yeah - there's a short circuit nature to this operation because it splits `part` first. so if you have `foo.bar.baz` as the path, it will evaluate if `foo` exists first, if it does then it will check for `bar` if that does not exist on `data.foo` then it will return the `default` value.